codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/customize/RequiredCustomizations.kt

```diff
@@ -69,5 +69,13 @@ class RequiredCustomizations : ClientCodegenDecorator {
         rustCrate.withModule(ClientRustModule.Types) {
             pubUseSmithyTypes(codegenContext, codegenContext.model)(this)
         }
+
+        val metaModule = when (codegenContext.settings.codegenConfig.enableNewCrateOrganizationScheme) {
+            true -> ClientRustModule.Meta
+            else -> ClientRustModule.root
+        }
```

Comment:
could add `CodegenContext.metaModule()` as a extension function since codegenContext _has_ codegen config you can fully contain the logic there

Reply:
Not a bad idea.